src/components/Cards/MeetingCard/MeetingCard.tsx

```diff
@@ -1,11 +1,14 @@
 import React from "react";
+import styled from "@emotion/styled";
 import { TAG_CONNECTOR } from "../../../constants/StyleConstants";
 import "@mozilla-protocol/core/protocol/css/protocol.css";
 import { strings } from "../../../assets/LocalizedStrings";
 
 export type MeetingCardProps = {
-  /** The poster image src of the event */
-  imgSrc: string;
+  /** The static poster image src of the event */
+  staticImgSrc: string;
+  /** The animated post image src of the event */
```

Comment:
"animated post image" --> "animated poster image" (???)

_Just comparing against the above comment ("static poster image") and seeing differences._